An E275 in functionnal tests prevents from building the project

The Python 3.8 build ubuntu-latest fails with:

```
./tests/functional/features/steps/common.py:336:11: E275 missing whitespace after keyword
```

Fixed by the obvious patch bellow:

``` diff
--- a/tests/functional/features/steps/common.py
+++ b/tests/functional/features/steps/common.py
@@ -333,7 +333,7 @@ def step_process_jlink_connected(context):
     process.start()
     process.join()
 
-    assert(all(queue.get() for _ in range(2)))
+    assert all(queue.get() for _ in range(2))
```

--- a/tests/functional/features/steps/common.py
+++ b/tests/functional/features/steps/common.py
@@ -333,7 +333,7 @@ def step_process_jlink_connected(context):
     process.start()
     process.join()
 
-    assert(all(queue.get() for _ in range(2)))
+    assert all(queue.get() for _ in range(2))
 
 
 @behave.then('I should not be able to open a new connection to it')

